src/app/notes/state/notes.selectors.ts

```diff
@@ -1,7 +1,7 @@
 import { createFeatureSelector, createSelector } from '@ngrx/store';
 import { NotesState } from '../interface/note';
 
-export const selectNotesState = createFeatureSelector<NotesState>('notes');
+const selectNotesState = createFeatureSelector<NotesState>('notes');
```

Comment:
No need to expose this selector since it is not used by any component. This also encourages the use of the more specific selectors